src/main/java/duke/list/List.java

```diff
@@ -0,0 +1,92 @@
+package duke.list;
+
+import duke.DukeException;
+
+public class List {
+
+    private static Task[] items;
+    private int size;
+
+    public List(){
+        items = new Task[100];
+        size = 0;
+    }
+
+    public void queryItems(){
+        printLine();
+        System.out.println("Here are the tasks in your list:");
+        for(int i=0; i<size; i++) {
```

Comment:
Try putting spaces in between to improve readability like so: 
`        for (int i = 0; I < size; i++) {
`